### PR TITLE
Realized EGO active abilities

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

Adds abilities to most/all realized EGO suits.

- Grinder allows its user to do the dash(teleport) to a nearby location, damaging everyone caught in the path.

## Why It's Good For The Game

More utility for the unique armor sets, instead of simply being better armor stat-wise.

### CODE REVIEW

**Comment by dsmith328** [comment]
okay but force the user to flip

**Comment by Kitsunemitsu** [comment]
I agree with redacted.

File: code/modules/spells/ability_types/realized_ego.dm [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		var/obj/effect/temp_visual/decoy/D = new /obj/effect/temp_visual/decoy(T, user)
+		var/matrix/M = matrix(D.transform)
+		M.Turn(45 * i)
+		D.transform = M
+		D.alpha = min(150 + i*15, 255)
+		animate(D, alpha = 0, time = 2 + i*2)
+		playsound(D, "sound/abnormalities/helper/move0[pick(1,2,3)].ogg", rand(10, 30), 1, 3)
+		for(var/obj/machinery/door/MD in T.contents)
+			if(MD.density)
+				addtimer(CALLBACK (MD, .obj/machinery/door/proc/open))
```
Comment (dsmith328) [inline code/modules/spells/ability_types/realized_ego.dm:41]: Finally, openable blast-doors.

Comment (EgorDinamit) [reply code/modules/spells/ability_types/realized_ego.dm:41]: Absolutely intended.

**Pull request merged**